api/handler.go

```diff
@@ -133,6 +133,10 @@ func (h *apiHandler) viewershipHandler() chi.Router {
 		With(h.cache(false)).
 		MethodFunc("GET", fmt.Sprintf(`/{%s}/total`, assetIDParam), h.getTotalViews)
 
+	// realtime viewership server side (internal-only)
+	h.withMetrics(router, "query_realtime_server_viewership").
+		MethodFunc("GET", "/internal/server/now", h.queryRealtimeServerViewership())
```

Comment:
Should we add something into the path to indicate viewership? e.g (bit verbose though) `/internal/server/viewership/now`

Reply:
the viewership handler is already defined as `/views`, so the full path will be `/data/views/internal/server/now`.

Reply:
Ah gotcha, I'd missed that - LGTM then